MyOnboarding/test/MyOnboardingApp.Api.Tests/Utils/ItemEqualityComparer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using MyOnboardingApp.Contracts.Models;
+
+namespace MyOnboardingApp.Tests.Utils
+{
+    public sealed class ItemEqualityComparer : IEqualityComparer<TodoListItem>
```

Comment:
I suggest hiding comparer in an extension method that would be called instead of `.Using(_comparer)`. You can stop caring about comparer's instance in tests and, as a bonus, you can rename the `Utils` folder to `Extensions` 😉